components/script/dom/canvasrenderingcontext2d.rs

```diff
@@ -371,20 +371,17 @@ impl CanvasRenderingContext2D {
                 None => return Err(Error::InvalidState),
             };
 
-            let (sender, receiver) = ipc::channel(self.global().time_profiler_chan().clone()).unwrap();
             let msg = CanvasMsg::Canvas2d(Canvas2dMsg::DrawImageInOther(
-                self.ipc_renderer.clone(),
                 self.get_canvas_id(),
                 image_size,
                 dest_rect,
                 source_rect,
-                smoothing_enabled,
-                sender),
+                smoothing_enabled
+                ),
```

Comment:
nit: This paren looks somewhat out of place. Either move the `DrawImageInOther(..` to the next line and indent the argumetns, or put this paren on the previous level and do `), context.get_canvas_id());`. Though not a terrible fan of that one :)

Reply:
Went with the first option.